File: DOHEE/Week1/명예의 전당 ( 1 ).py
# 10분
def solution(k, score):
    sortedScore = []
    answer = []
    for i in range(len(score)):
        sortedScore.append(score[i])
        sortedScore.sort(reverse = True)
        if i < k - 1:
            answer.append(sortedScore[i])
        else:
            answer.append(sortedScore[k - 1])
    return answer

File: DOHEE/Week1/명예의 전당 ( 1 ).js
// 20분
function solution(k, score) {
    var answer = [];
    const sortedScore = [];
    for (let i = 0; i < score.length; i++) {
        sortedScore.push(score[i]);
        sortedScore.sort((a, b) => b - a);
        if (i < k - 1) {
            answer.push(sortedScore[i]);
        } else {
            answer.push(sortedScore[k - 1]);
        }
    }
    return answer;
}
